python/pyarrow/tests/test_types.py

```diff
@@ -1263,17 +1266,20 @@ def test_field_modified_copies():
 
 def test_is_integer_value():
     assert pa.types.is_integer_value(1)
-    assert pa.types.is_integer_value(np.int64(1))
+    if np is not None:
+        assert pa.types.is_integer_value(np.int64(1))
     assert not pa.types.is_integer_value('1')
 
 
+@pytest.mark.numpy
```

Comment:
Can we just apply the same principle here and below (guard NumPy-specific checks with `if np is not None` instead of skipping the whole test if NumPy is absent)?